app/views/account/companies/show.html.slim

```diff
@@ -67,3 +67,18 @@
           .card-content
             .card-body
               = render 'account/companies/html/social_links', company: @company
+        - if @twitter.tweets.any?
+          .card
+            .card-header
+              .row
+                .col-md-6
+                  h4.card-title= "#{@company.name} tweets"
+                  h5.card-title
+                    =link_to @company.twitter, target: '_blank' do
+                      | @#{@twitter.screen_name}

```

Comment:
screen name may be `nil`!

Reply:
but if `screen_name` is `nil` there will be errors raised by twitter and this whole part won't be rendered